lib/content/package-json.hbs

```diff
@@ -42,7 +42,10 @@
   "standard": {{{ del }}},
   "tap": {
     {{#if workspacePaths}}
-    "test-ignore": "^({{ join workspacePaths "|" }})/",
+      "exclude": {{#if tap18}}[
+        "{{ join workspaceGlobs "," }}"
+        ]{{else }}{{{ del }}}{{/if}},
```

Comment:
🤔 For repos that have prettier enabled, I wonder if prettier will want the newlines for brackets removed, possibly depending on line length.

Reply:
Not in the package.json it won't.  It better not.